pom.xml

```diff
@@ -53,6 +54,12 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>-XX:MaxPermSize=${permgen.size}</argLine>
```

Comment:
Thanks, this should be fixed now by overriding the ```${argLine}``` property from parent pom instead of the entire surefire configuration.